src/main/java/lotto/domain/factory/LottoMakeAble.java

```diff
@@ -0,0 +1,8 @@
+package lotto.domain.factory;
+
+import lotto.domain.Lotto;
+
+public interface LottoMakeAble {
+
+	public abstract Lotto create();
```

Comment:
```suggestion
	Lotto create();
```

인터페이스의 경우 public abstract를 생략할 수 있습니다!

Reply:
앞으론 생략하도록 하겠습니다!
감사합니다 :)